apps/vanilla/src/pages/index.css

```
@@ -1,5 +1,302 @@
 @import url("../global/index.css");
 
-.title {
-  color: var(--heading-color-1);
+/* Font */
+.font-subtitle {
+  font-size: var(--font-size-lg);
+  line-height: var(--line-height-md);
+  font-weight: var(--font-weight-medium);
+  text-transform: uppercase;
+}
+
+.font-body {
+  font-size: var(--font-size-xl);
+  line-height: var(--line-height-lg);
+  font-weight: var(--font-weight-regular);
+}
+
+.font-body-small {
+  font-size: var(--font-size-md);
+  line-height: var(--line-height-md);
+  font-weight: var(--font-weight-medium);
+}
+
+.font-logo {
+  font-size: var(--font-size-xl);
+  line-height: var(--line-height-md);
+  font-weight: var(--font-weight-bold);
+}
+
+/* Container */
+.container {
+  --container-width: 1244px;
+
+  max-width: var(--container-width);
+  margin: 0 auto;
+  display: grid;
+  gap: var(--spacing-md);
+}
+
+.header {
+  color: var(--on-surface-color);
+  height: var(--header-height);
+  border-bottom: 1px solid var(--border-color);
+  display: flex;
+  align-items: center;
+}
+
+.main-section__books-wrapper {
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  gap: var(--spacing-lg);
+}
+
+.main-section {
+  display: flex;
+  gap: var(--spacing-lg);
+}
+
+/* Books Group */
+.books-wrapper__books-group {
+  display: grid;
+  grid-template-columns: repeat(5, 1fr);
+  gap: var(--spacing-sm);
+}
+
+.books-group__book-card {
+  --book-width: 167px;
+
+  width: var(--book-width);
+  display: flex;
+  flex-direction: column;
+  gap: var(--spacing-sm);
+}
+
+.book-card__cover {
+  aspect-ratio: 2/3;
+  border-radius: var(--border-radius-lg);
+  overflow: hidden;
+  border: 1px solid var(--border-color);
+}
+
+.truncate {
+  display: -webkit-box;
+  -webkit-box-orient: vertical;
+  -webkit-line-clamp: 2;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+.books-wrapper__books-year {
+  position: sticky;
+  top: 0;
+  background-color: var(--surface-color);
+  padding: var(--spacing-sm) 0;
+}
+
+.cover__img {
+  width: 100%;
+  height: 100%;
+}
+
+/* Filter */
+.filters {
+  width: var(--filter-size-lg);
+  display: flex;
+  flex-direction: column;
+  gap: var(--spacing-lg);
+  position: relative;
+}
+
+.filters__form-wrapper {
+  position: fixed;
+  width: var(--filter-size-lg);
+}
+
+.form-wrapper__form {
+  display: grid;
+  grid-gap: var(--spacing-lg);
+}
+
+.form-control__input {
+  width: 100%;
+  height: var(--form-control-height);
+  background-color: transparent;
+  border: none;
+}
+
+.form__checkbox {
```

Checkbox should be the same naming with other element
```suggestion
.form-control__checkbox {
```